locator/guice/src/main/java/org/eclipse/kapua/locator/guice/KapuaLocatorImpl.java

```diff
@@ -117,7 +117,10 @@
         for (Binding<?> binding : bindings.values()) {
             final Class<?> clazz = binding.getKey().getTypeLiteral().getRawType();
             if (KapuaService.class.isAssignableFrom(clazz)) {
-                servicesList.add(injector.getInstance(clazz.asSubclass(KapuaService.class)));
+                KapuaService serviceInstance = injector.getInstance(clazz.asSubclass(KapuaService.class));
```

Comment:
Set looks good but yeah I also prefer not to change for the same reasons.
Will check with @Coduz and @lorthirk a do it later.